cmd/background-controller/main.go

```diff
@@ -138,11 +138,8 @@ func main() {
 	}
 	eventGenerator := event.NewEventGenerator(
 		setup.KyvernoDynamicClient,
-		kyvernoInformer.Kyverno().V1().ClusterPolicies(),
-		kyvernoInformer.Kyverno().V1().Policies(),
-		maxQueuedEvents,
```

Comment:
We want to remove this flag if `maxQueuedEvents` is not needed anymore.
```
flagset.IntVar(&maxQueuedEvents, "maxQueuedEvents", 1000, "Maximum events to be queued.")
```

Reply:
And to remove from the doc as well.

Reply:
We will definitely have to deprecate and/or remove this at some point, it's too early to do that now IMO.

Reply:
With this change, the flag `maxQueuedEvents` will be useless, and we don't want to lose track of this removal. What could be the best way to to coordinate the doc changes?

Reply:
We'll see, there's a chance we will introduce a queue at some point.

Reply:
Can we log an issue to record the current state - the flag is of no use? We can update if we introduce new behaviors.

Reply:
I'll do.